dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -0,0 +1,471 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
suggestion: this file also seems very long. may be we can keep the recorder and carry Profile and FrameMetrics to another common or util class

Reply:
This repository, unlike flutter/engine, does not use part files, and so one Dart file is also one library. I wouldn't break it up because these classes are closely related to the other classes in this library and are exported by one library. Otherwise, we'd need to re-export them, or have call sites import multiple libraries. Hopefully this file won't grow that much because most of the code will be in the many many benchmarks that we'll write, each of which will be quite small.

I also grepped the code in this repo and found 724 files that are longer than this file, so the current line count is not at all excessive by Flutter standards.